service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/SampleRoboticsTestDataHelper.java

```diff
@@ -61,7 +61,7 @@ public static RoboticsData invalidRoboticsDataWithZeroOwningCourt() {
                     .postcode("BT22 2345")
                     .country("UK")
                     .build())
-                .relationshipToChild(null)
+                .relationshipToChild("Cousin")
```

Comment:
Necessary change?

Reply:
yes, relationshipToChild is mandatory field and will never be null. So this is unrealistic scenario

Reply:
Ahh I see you have updated it in the populated case details